terraform/gcp/external-loadbalancer.tf

```diff
@@ -0,0 +1,103 @@
+## CDN external load balancer
+module "medplum-lb-https" {
+  source                          = "GoogleCloudPlatform/lb-http/google"
+  version                         = "~> 11.1"
+  name                            = "medplum-elb"
+  project                         = var.project_id
+  firewall_networks               = [module.vpc.network_name]
+  target_tags                     = []
+  url_map                         = google_compute_url_map.cdn_url_map.self_link
+  create_url_map                  = false
+  ssl                             = true
+  managed_ssl_certificate_domains = ["${var.user_content_domain}", "${var.static_asset_domain}"]
+
+  backends = {
+    default = {
+      protocol    = "HTTP"
+      port        = 80
+      port_name   = "http"
+      timeout_sec = 10
+      enable_cdn  = true
+
+      cdn_policy = {
+        cache_mode  = "CACHE_ALL_STATIC"
```

Comment:
please change to `USE_ORIGIN_HEADERS` so it uses cache headers from the bucket file metadata

Reply:
done